tests/integration/fixtures/mini_sentry.py

```diff
@@ -119,12 +119,21 @@ def basic_project_config(self, project_id, dsn_public_key=None):
             },
         }
 
-    def add_basic_project_config(self, project_id, dsn_public_key=None):
+    def add_basic_project_config(self, project_id, dsn_public_key=None, extra=None):
         ret_val = self.basic_project_config(project_id, dsn_public_key)
+
+        if extra:
+            extra_config = extra.get("config", {})
+            ret_val = {
+                **ret_val,
+                **extra,
+                "config": {**ret_val["config"], **extra_config},
+            }
+
         self.project_configs[project_id] = ret_val
         return ret_val
 
-    def add_full_project_config(self, project_id, dsn_public_key=None):
+    def full_project_config(self, project_id, dsn_public_key=None, extra=None):
```

Comment:
This is fine, but don't we usually just patch this straight away in the tests?

Reply:
yes... but it is a bit odd to go inside mini_sentry's bowels to change an "already configured" project.

Reply:
yes we call `add_full_project_config` and then mess around in mini_sentry's bowels. 
I think that changing the project_config from inside mini_sentry is unnecessarily ugly.